pyodide-build/pyodide_build/cli/build.py

```diff
@@ -67,37 +67,40 @@ def url(
     backend_flags = ctx.args
     curdir = Path.cwd()
     (curdir / "dist").mkdir(exist_ok=True)
-    with requests.get(package_url, stream=True) as response:
```

Comment:
```
        response = cast(requests.Response, response)  # placate mypy
```

Reply:
yes, I agree it's probably better and pre-commit passes in https://github.com/pyodide/pyodide/pull/3609

So maybe let's go with that, if no objections from @ryanking13 ?

Reply:
Indeed, that's better. Thanks @cclauss!